src/components/RefineSearch/CancelSubmitButtonGroup.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, Icon, ButtonGroup } from "@nypl/design-system-react-components"
+import type { SyntheticEvent } from "react"
+
+interface SearchButtonProps {
+  cancelHandler: (e: SyntheticEvent) => void
+  submitLabel: string
+  cancelLabel: string
+  formName: string
+  disableSubmit?: boolean
+}
+
+const SearchButtons = ({
```

Comment:
Rename this the same as the file. It's a bit confusing otherwise unless if this should be `<SearchButtons />` instead.